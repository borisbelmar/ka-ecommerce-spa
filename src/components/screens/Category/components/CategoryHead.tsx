import ContainerLayout from "@/components/layouts/ContainerLayout"

interface CategoryHeadProps {
  name: string
}

export default function CategoryHead ({ name }: CategoryHeadProps) {
  return (
    <div className="bg-indigo-500 text-white">
      <ContainerLayout className="py-24">
        <h1 className="font-semibold text-3xl">
          Productos en la categor√≠a <span className="uppercase font-bold">{name}</span>
        </h1>
        <p className="text-lg opacity-80 mt-2">
          Lorem ipsum dolor sit
          amet consectetur adipisicing elit. Quisquam, voluptatum
        </p>
      </ContainerLayout>
    </div>
  )
}