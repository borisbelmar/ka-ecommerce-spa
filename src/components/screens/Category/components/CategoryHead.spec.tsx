import { render, screen } from '@testing-library/react'
import CategoryHead from './CategoryHead'

describe('CategoryHead', () => {
  it('should render the category name', () => {
    const mockName = 'mock category name'

    render(<CategoryHead name={mockName} />)

    expect(screen.getByText(mockName)).toBeInTheDocument()
  })
  it('should match snapshot', () => {
    const mockName = 'mock category name'

    const { container } = render(<CategoryHead name={mockName} />)

    expect(container).toMatchSnapshot()
  })
})