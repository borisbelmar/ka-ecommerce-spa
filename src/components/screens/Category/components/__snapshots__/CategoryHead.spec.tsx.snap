// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryHead should match snapshot 1`] = `
<div>
  <div
    class="bg-indigo-500 text-white"
  >
    <section
      class="w-full mx-auto max-w-screen-xl px-8 items-center justify-between py-24"
    >
      <h1
        class="font-semibold text-3xl"
      >
        Productos en la categor√≠a 
        <span
          class="uppercase font-bold"
        >
          mock category name
        </span>
      </h1>
      <p
        class="text-lg opacity-80 mt-2"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum
      </p>
    </section>
  </div>
</div>
`;
